bindings/python/test/environment/objects/celestial/test_earth.py

```diff
@@ -0,0 +1,201 @@
+
+    earth: Earth = Earth.default()
+
+    assert earth is not None
+    assert isinstance(earth, Earth)
+
+################################################################################################################################################################
+
+def test_earth_static_EGM2008_constructor ():
+
+    earth: Earth = Earth.EGM2008()
+
+    assert earth is not None
+    assert isinstance(earth, Earth)
+
+    earth_custom: Earth = Earth.EGM2008(2190, 2160)
+
+    assert earth_custom is not None
+    assert isinstance(earth_custom, Earth)
+
+################################################################################################################################################################
+
+def test_earth_static_EGM96_constructor ():
+
+    earth: Earth = Earth.EGM96()
+
+    assert earth is not None
+    assert isinstance(earth, Earth)
+
+    earth_custom: Earth = Earth.EGM96(360, 360)
+
+    assert earth_custom is not None
+    assert isinstance(earth_custom, Earth)
+
+################################################################################################################################################################
+
+def test_earth_static_EGM84_constructor ():
+
+    earth: Earth = Earth.EGM84()
+
+    assert earth is not None
+    assert isinstance(earth, Earth)
+
+    earth_custom: Earth = Earth.EGM84(180, 180)
+
+    assert earth_custom is not None
+    assert isinstance(earth_custom, Earth)
+
+################################################################################################################################################################
+
+def test_earth_static_WGS84_EGM96_constructor ():
+
+    earth: Earth = Earth.WGS84_EGM96()
+
+    assert earth is not None
+    assert isinstance(earth, Earth)
+
+    earth_custom: Earth = Earth.WGS84_EGM96(180, 180)
+
+    assert earth_custom is not None
+    assert isinstance(earth_custom, Earth)
+
+################################################################################################################################################################
+
+def test_earth_static_WGS84_constructor ():
+
+    earth: Earth = Earth.WGS84()
+
+    assert earth is not None
+    assert isinstance(earth, Earth)
+
+    earth_custom: Earth = Earth.WGS84(20, 0)
+
+    assert earth is not None
+    assert isinstance(earth_custom, Earth)
+
+################################################################################################################################################################
+
+def test_earth_static_spherical_constructor ():
+
+    earth: Earth = Earth.spherical()
+
+    assert earth is not None
+    assert isinstance(earth, Earth)
+
+################################################################################################################################################################
+
+def test_earth_static_properties_EGM2008 ():
+
+    assert EGM2008.gravitational_parameter is not None
```

Comment:
These are left as `is not None` because the values are hard coded in the Earth.cpp file so I don't think it makes sense to have them hardcoded here in a unit test.